tle/tle-application/src/main/kotlin/com/example/tleapplication/application/news/NewsController.kt

```diff
@@ -0,0 +1,155 @@
+package com.example.tleapplication.application.news
+
+import com.example.tleapplication.domain.news.Category
+import com.example.tleapplication.domain.news.NewsService
+import com.example.tleapplication.support.logging.TraceIdResolver
+import com.example.tleapplication.support.response.TleApiResponse
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.Valid
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@RestController
+@RequestMapping("/news")
+@Tag(name = "뉴스 API", description = "뉴스 API")
+class NewsController(
+    private val newsService: NewsService,
+    private val traceIdResolver: TraceIdResolver
+) {
+    @Operation(hidden = true)
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    fun registerNews(
+        @Valid @RequestBody request: CreateNewsRequest
+    ): TleApiResponse<Unit?> {
+        val news = request.toDomain()
+        newsService.registerNews(news)
+        return TleApiResponse.success(
+            traceIdResolver.getTraceId(),
+            HttpStatus.CREATED,
+            null
+        )
+    }
+
+    @Operation(
+        summary = "뉴스 조회",
+        description = "뉴스 조회 API",
+        responses = [
+            ApiResponse(responseCode = "200", description = "뉴스 조회 성공"),
+            ApiResponse(responseCode = "500", description = "Internal Server Error", content = arrayOf(
+                Content(schema = Schema(hidden = true))
+            )),
+        ],
+    )
+    @GetMapping("/{news-id}")
+    @ResponseStatus(HttpStatus.OK)
+    fun getNews(
+        @PathVariable("news-id") id: Long
+    ): TleApiResponse<NewsResponse> {
+        val news = newsService.getNews(id)
+        return TleApiResponse.success(
+            traceIdResolver.getTraceId(),
+            HttpStatus.OK,
+            NewsResponse.from(news)
+        )
```

Comment:
```suggestion
        return TleApiResponse.success(
            traceId = traceIdResolver.getTraceId(),
            status = HttpStatus.OK,
            body = NewsResponse.from(news)
        )
```
클래스나 메소드 호출에서 파라미터가 2개 이상일때 다음과 같이 명시하면 가독성이 더 좋습니다. 저는 코틀린에서 이렇게 작성하는게 습관이라 추천드려요..!